linode/firewalls/framework_schema.go

```diff
@@ -0,0 +1,153 @@
+package firewalls
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/linode/terraform-provider-linode/linode/helper"
+	"github.com/linode/terraform-provider-linode/linode/helper/frameworkfilter"
+)
+
+var filterConfig = frameworkfilter.Config{
+	"id":    {APIFilterable: true, TypeFunc: helper.FilterTypeInt},
+	"label": {APIFilterable: true, TypeFunc: helper.FilterTypeString},
+	"tags":  {APIFilterable: true, TypeFunc: helper.FilterTypeString},
+
+	"status": {APIFilterable: false, TypeFunc: helper.FilterTypeString},
```

Comment:
`created` and `updated` are also API filterable
https://www.linode.com/docs/api/networking/#firewalls-list__responses